### DISCUSSION TITLE
Nested tree and intls

### DISCUSSION THREAD

**Discussion by ghost** [sveltejs/kit#8950] [General]
Setting up a site covering three countries and exploring options in SvelteKit to use a single domain.

Trying to achieve:

domain.com point to root page.svelte
domain.com/es point to the same page.svelte (with an intl flag=es)
domain.com/fr point to the same page.svelte (with an intl flag=fr)

same logic for domain.com/blog, domain.com/es/blog, domain.com/fr/blog

Is there a way in hooks or other to pickup the intl path (eg es and fr above) but in the folder structure bypass them so that all three points to same?

Apologies if not clear, and if Im missing something obvious. Want to avoid having to have duplicate trees and Im sure SvelteKit is too smart to have that.

Thanks

**Comment by david-plugge** [comment]
Use an optional parameter
https://kit.svelte.dev/docs/advanced-routing#optional-parameters

**Reply by ghost** [reply]
Thank you!
